src/installation/guides/chroot.md

```diff
@@ -0,0 +1,235 @@
+
+If unsure what partitions to create, create (1) 1GB in size with type `EFI
+System` (code EF00) (2) remainder of the drive with type `Linux Filesystem`
+(code 8300).
+
+Then format these partitions with fat32 and ext4 respectively.
+
+```
+# mkfs.vfat /dev/sda1
+# mkfs.ext4 /dev/sda2
+```
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `dev/sda1` is the EFI
+System Partition
+
+```
+# mount /dev/sda2 /mnt/
+# mkdir -p /mnt/boot/efi/
+# mount /dev/sda1 /mnt/boot/efi/
+```
+
+Initialize swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution.
+
+> Note: musl binaries are not available for i686.
+
+For example, this is the url for glibc from the Tier 1 server in Finland,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Use xbps to boot strap the distribution by installing the `base-system`
+metapackage.
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+## Configuration
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount -t proc /proc /mnt/proc
+# mount -t sysfs /sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the DNS configuration into the new root so that xbps can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+Write the hostname to `/mnt/etc/hostname`. Go through the options in
+[`/mnt/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/mnt/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
```

Comment:
I think it's already fairly explicit, "for glibc builds". A whole lot of warning isn't necessary because if you do accidentally do that on a musl build, it won't harm anything anyway.